utk/js/sale.js

```diff
@@ -0,0 +1,70 @@
+//$(document).ready(function() {
+$( window ).on( "load", function(){
```

Comment:
I just want to make sure we are only running this on pages that need the JS markup, if we need this at all. I'm wondering if we can't just have the .net platform update the prices for us and apply the correct classes. 

If we do this,  I want to come in here and remove this file.

Reply:
Yes, I agree. It is in there just incase i will remove it once we get a response